Source/sdc/Castro_sdc_util.H

```diff
@@ -170,9 +168,13 @@ sdc_update_o2(const int i, const int j, const int k,
 
         sdc_solve(dt_m, U_old, U_new, C_zone, sdc_iteration);
 
-        // we solved our system to some tolerance, but let's be sure we are conservative by
-        // reevaluating the reactions and { doing the full step update
-        single_zone_react_source(U_new, R_full, burn_state);
+        // we solved our system to some tolerance, but let's be sure
+        // we are conservative by reevaluating the reactions and {
```

Comment:
superfluous {

Reply:
why?
The idea is that we might have done it one a little bit at a time, and next we want to do it in terms of averages and as a single update

Reply:
oh... I see what you mean ;)